heartbeat/docker-compose

```diff
@@ -0,0 +1,297 @@
+#!/bin/bash
```

Comment:
You need to rename it to `docker-compose.in` and update this to `#!@BASH_SHELL@` and add it to configure.ac for it to detect the bash path properly:
https://github.com/ClusterLabs/resource-agents/blob/master/configure.ac#L1005

Or change it to /bin/sh if all the code is